pkg/types/builder.go

```diff
@@ -22,11 +22,12 @@ import (
 	"go/types"
 	"sort"
 
-	"github.com/crossplane/crossplane-runtime/pkg/fieldpath"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	twtypes "github.com/muvaf/typewriter/pkg/types"
 	"github.com/pkg/errors"
 
+	"github.com/crossplane/crossplane-runtime/pkg/fieldpath"
```

Comment:
We group these with the upper block since it's in a different github org, at least for now.

Reply:
Thanks @muvaf. That's my IDE kicking in.